scylla-cdc-replicator/src/replicator_consumer.rs

```diff
@@ -91,7 +92,44 @@ impl ReplicatorConsumer {
         Ok(())
     }
 
-    // Recreates INSERT/DELETE statement on non-frozen map.
+    // Function replicates adding and deleting elements
+    // using v = v + {} and v = v - {} cql syntax to non-frozen map.
+    // Chained operations are supported.
+    async fn update_map_elements<'a>(
+        &self,
+        column_name: &str,
+        data: &'a CDCRow<'_>,
+        values_for_update: &mut Vec<&'a CqlValue>,
+        timestamp: i64,
+    ) -> anyhow::Result<()> {
+        let empty_map = Map(vec![]); // We have to declare it, because we can't take a reference to tmp value in unwrap.
+        let value = data.get_value(column_name).as_ref().unwrap_or(&empty_map);
+        // Order of values: ttl, added elements, pk condition values.
+
+        let deleted_set = Set(Vec::from(data.get_deleted_elements(column_name)));
+        let mut values_for_update: Vec<&CqlValue> = values_for_update.clone();
```

Comment:
That's a shame we have to clone here... However optimizing this will require a refactor too large for this PR, so I guess it's OK for now. It would be nice to do it eventually, but not until all planned replicator consumer features are implemented.

Reply:
(to be clear: the main priority is to get the replicator to work correctly)